editor/src/main/java/net/rpgtoolkit/editor/ui/MainWindow.java

```diff
@@ -456,11 +463,11 @@ public void checkFileExtension(File file) {
       addToolkitEditorWindow(EditorFactory.getEditor(openBoard(file)));
     } else if (fileName.endsWith(".ene") || fileName.endsWith(".ene.json")) {
       addToolkitEditorWindow(EditorFactory.getEditor(openEnemy(file)));
+    } else if (fileName.endsWith(".item") || fileName.endsWith("itm.json")) {

```

Comment:
Should we be using those file extension properties for this? That would be pretty wordy, but maybe more durable?